providers/http/tests/unit/http/sensors/test_http.py

```diff
@@ -317,8 +319,25 @@ def test_get_response_check(self):
         )
         op.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE, ignore_ti_state=True)
 
+    @pytest.mark.skipif(not AIRFLOW_V_3_0_PLUS, reason="Test only for Airflow 3.0+")
+    @mock.patch("airflow.providers.http.hooks.http.Session", FakeSession)
+    def test_sensor(self, run_task):
+        sensor = HttpSensor(
+            task_id="http_sensor_check",
+            http_conn_id="http_default",
+            endpoint="/search",
+            request_params={"client": "ubuntu", "q": "airflow", "date": "{{ds}}"},
+            headers={},
+            response_check=lambda response: f"apache/airflow/{DEFAULT_DATE:%Y-%m-%d}" in response.text,
+            poke_interval=5,
+            timeout=15,
+            dag=self.dag,
+        )
+        run_task(sensor)
```

Comment:
Should we assert something? I know we are testing if it works, so not sure

Reply:
Naah, just that there is no exception raised is enough -- also retain previous test behaviour